client/striplighttimer.py

```diff
@@ -0,0 +1,29 @@
+import requests
+import time
+import datetime
+headers={'accept': 'application/json'}
+payload={'power':'true'}
+mario_API="http://mario.clients.shimley:8000/v1/strip/1/power"
+luigi_API="http://luigi.clients.shimley:8000/v1/strip/1/power"
+def timer():
+    while True: 
+        
+        now = datetime.datetime.today().timetuple()
+        if now.tm_hour == 18 and now.tm_min == 0:
+            payload={'power':'true'}
+            m=requests.put(mario_API, headers=headers,params=payload)
+            l=requests.put(luigi_API, headers=headers,params=payload)
+        if now.tm_hour == 2 and now.tm_min == 0:
+            payload={'power':'false'}
+            m=requests.put(mario_API, headers=headers,params=payload)
+            l=requests.put(luigi_API, headers=headers,params=payload)
```

Comment:
```suggestion
            payload = {'power':'true'}
            mario = requests.put(url=MARIO_API, headers=HEADERS, params=payload)
            luigi = requests.put(url=LUIGI_API, headers=HEADERS, params=payload)
        elif now.tm_hour == 2 and now.tm_min == 0:
            payload = {'power':'false'}
            mario = requests.put(url=MARIO_API, headers=HEADERS, params=payload)
            luigi = requests.put(url=LUIGI_API, headers=HEADERS, params=payload)
        else:
            time.sleep(59)
            continue
```

Switching to an if/elif/else here will sleep and skip if there is nothing to do, preventing an error trying to evaluate `mario` and `luigi` which will not exist.  also cleaned up some variable names and styles.